barrantes/index.html

```diff
@@ -0,0 +1,176 @@
+            <img src="https://i.pinimg.com/564x/5d/29/93/5d2993b3ad0f1d37c630d3132114a1c0.jpg" />
+        </div>
+    </div>
+    <div class="contact">
+        <h2>Contact Me</h2>
+        <form>
+            <div class="email">
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" />
+            </div>
+            <div class="name-contact">
+                <label for="Name">Full Name:</label>
+                <input type="text" id="name-contact" name="Name" />
+            </div>
+            <div class="subject">
+                <label for="subject">Subject:</label>
+                <input type="text" id="subject" name="subject">
+            </div>
+            <div class="age">
+                <label for="age">Age:</label>
+                <input type="number" id="age" name="age">
+            </div>
+            <div class="copy">
+                <label for="copy">Send me a copy</label>
+                <input type="checkbox" name="copy" id="copy">
+            </div>
+            <div class="message">
+                <label for="message">Message:</label>
+                <textarea id="message" name="message" rows="5" cols="33">
+                </textarea>
+            </div>
+            <div class="submit">
+                <button type="submit">Submit</button>
+            </div>
+        </form>
+    </div>
+    <div class="goals">
+        <div class="split-right">
+            <div class="section-container">
+                <div class="slide">
+                    <h2>My Goals</h2>
+                    <ul>
+                        <li>To graduate</li>
+                        <li>Receive a job offer that is good for a new grad</li>
+                        <li>Pass the landerized exam</li>
+                        <li>Good health and stable financially</li>
+                    </ul>
+                    <h2>Comment</h2>
+                    <ul>
+                        <li> May your come true! -Precious </li>
+                        <li> Great Job! - Eliza</li>
+                        <li>Goodluck on your goals. :D - Mikaella</li>
+                        <li>Praying for you! - Alona</li>
+                    </ul>
+                </div>
+                <div class="comm">
+                    <h2>Comment Form</h2>
+                    <form id="comments_form">
+                        <div class="fname">
+                            <label for="fname">Name: </label>
+                            <input type="text" name="fname" id="fname">
+                        </div>
+                        <div class="com">
+                            <label for="comment">Comment: </label>
+                            <textarea id="comment" name="comment" rows="5" 
+                            cols="33"></textarea>
+                        </div>
+                        <div class="com-b">
+                            <button disabled type="submit" id="com-b">Submit
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
                            <button disabled type="submit" id="com_b">Submit
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)